src/versionedRouter.js

```diff
@@ -527,6 +527,11 @@ async function routerHandleDest(ctx) {
       statTags: errObj.statTags,
     };
 
```

Comment:
`routerHandleDest` internally calls `processRouterDest` at https://github.com/rudderlabs/rudder-transformer/blob/20f76c4a703dad53c526b662ec7f6410cc37e874/src/versionedRouter.js#L481 which invokes `handleRtTfSingleEventError` that's already adding `authErrorCategory` to the response.
So, what flow/usecase is forcing to have this again here ?

Reply:
If someone has actually implemented `processRouterDest` by themselves wherever they don't have `try-catch` if some error is thrown, it will hit this catch block & if the error has `authErrorCategory` we would then send proper `authErrorCategory` information to server correctly